### ISSUE TITLE
Get TLS certificate in place

### ISSUE THREAD

**Issue by konklone** [18F/foia-hub#547]
Purchase a TLS certificate, upload it to Amazon Web Services, and have our Cloud Foundry set up use it to serve the application.

**Comment by konklone** [comment]
This will be done strictly in Amazon Web Services, used by an ELB (in the CF VPC) that we configure ourselves. Cloud Foundry itself won't play a role in the HTTPS termination.

**Comment by konklone** [comment]
I've purchased the certificate for `open.foia.gov`, using SSLMate's [DNS-based validation](https://github.com/SSLMate/sslmate/issues/9). This did not require setting up any MX records or receiving emails. It did, as part of the process, create a CNAME record with a hash as the value, to validate domain ownership. 

While the CNAME record is not required to continue validating the domain (the way it would for e.g. Google Webmaster Tools), there's no reason to delete it either, and future domain issuances will automatically delete it during the reissuance process. 

I've uploaded the key and cert to AWS us-east-1 as `open-foia-gov`, and deleted the materials from the machine I generated it on. If we ever need to generate it again during the year of time we bought, we can use `sslmate reissue`, which will not cost any money (but will use the same start date, 2015-03-23) and will revalidate the domain.
